Add note on finding templates (#2)
bemosior/wardley-mapping-canvas: README.md

1. Open an issue here, describing what you're aiming to improve (we'll have some helpful discussion)
2. Register an account on [RealtimeBoard](https://realtimeboard.com)

> 3. Create a new board (Note: this must be [from the dashboard](https://realtimeboard.com/app/dashboard/) to use a template)

Reviewer: What do you think about this wording? It puts the helpful link you added up-front, so they are tempted to just click it and get moving.
```suggestion
3. Open [the dashboard](https://realtimeboard.com/app/dashboard/) and create a new board
```

Author: It's much more concise. LGTM 👍

Reviewer: Thank you!
